src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.SECURITY_ATTRIBUTES.cs

```diff
@@ -15,6 +15,7 @@ internal struct SECURITY_ATTRIBUTES
             internal uint nLength;
             internal IntPtr lpSecurityDescriptor;
             internal BOOL bInheritHandle;
+            internal unsafe byte* pSecurityDescriptor;
```

Comment:
Brought this from framework. The only difference is that it does not assign null by default. Should I?

Reply:
This structure is defined by the Windows SDK. The [documentation](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/aa379560(v=vs.85)) does not say anything about `pSecurityDescriptor` field that you are adding. What is this field for?

Reply:
Yeah, don't add this. Assign to `lpSecurityDescriptor` as I've described below.

Reply:
Thanks for noticing. Yes, I'll remove it.